element-hq/element-x-android
# Block opening space

## Type of change


## Content


It can happen, when using permalink that a Space become accessible to Element X user.
The PR ensure that the user is routed to a Space preview with an clear message that Space are not supported yet

<img width="333" alt="image" src="https://github.com/element-hq/element-x-android/assets/3940906/e0d4ed9e-2269-4dd3-a87d-a5572879bb02">

Other changes:
- ensure that `roomId` will not be rendered on screen, and add some screenshot test around it
- ensure that room default avatar will not use `!` (from the roomId), but fallback to `#` in this case.

## Motivation and context



## Screenshots / GIFs



See recorded once, when this will appear.

## Tests



- Join a space
- send a permalink to this space to a room
- click on this permalink -> previously the space timeline was rendered, no the screen above is displayed.

## Tested devices

- OS version(s):

## Checklist
